src/gwt/src/org/rstudio/studio/client/workbench/views/buildtools/BuildPresenter.java

```diff
@@ -343,29 +346,36 @@ private void executeBuild(final String type, final String subType)
    {
       // attempt to start a build (this will be a silent no-op if there
       // is already a build running)
+      final String caption = "Build";
       workbenchContext_.setBuildInProgress(true);
```

Comment:
Seems like we'll get stuck in the "build in progress" state if the user decides not to proceed with the build after seeing that they have busy terminals?